packages/eslint-plugin-next/lib/rules/no-head-element.js

```diff
@@ -14,7 +14,10 @@ module.exports = {
   create: function (context) {
     return {
       JSXOpeningElement(node) {
-        if (node.name.name !== 'head') {
+        const paths = context.getFilename()
```

Comment:
Is `context.getFilename()` absolute? If so we may need to check this differently, possibly we want to check if it includes `app` but not `pages` in the path.